api/app/signals/apps/email_integrations/actions/abstract.py

```diff
@@ -105,8 +105,12 @@ def send_mail(self, signal, dry_run=False):
             return 0  # No mail sent, return 0. Same behaviour as send_mail()
 
         subject, message, html_message = self.render_mail_data(context)
+
+        if len(recipient_list) == 0:
```

Comment:
Deze logica vervangen met een nieuwe method voor het ophalen van de recipient_list

```
recipient_list = self.get_recipients(signal)

def get_recipient_list(self, signal):
    return [signal.reporter.email]
```
De recipient_list method kan dan in de AssignAction worden overschreven zodat het geen inpact heeft op het core systeem van alle emails

De recipient_list kan dan via de kwargs van de system_mail method worden toegevoegt.

Reply:
Uiteindelijk suggestie van @vanbuiten overgenomen: https://github.com/Amsterdam/signals/pull/1057#discussion_r962579379